OpenOversight/app/templates/label_data.html

```diff
@@ -100,52 +100,53 @@ <h2 class="horizontal-padding font-weight-300">How you can help</h2>
               <p class="lead margin-less">Sort Images</p>
               <div class="font-weight-300">Sort submitted images into those with and without officers.</div>
             </div>
-          <div class="col-lg-4 horizontal-padding""> <span class="fa fa-users fa-2x"></span>
```

Comment:
Addressed the double `""` at the end of the `div` above this message, so there was a lot of formatting done around it.